activestorage/active.py

```diff
@@ -10,6 +10,42 @@
 from activestorage import netcdf_to_zarr as nz
 
 
+def uri_analyzer(uri):
```

Comment:
Don't we want to assess if active storage reductions are possible, not just that the file exists?

Reply:
yes! But how do we do that? We need a set of standards here to get conditions in

Reply:
See below ....